src/platforms/linux/daemon/dbusservice.h

```diff
@@ -34,16 +35,19 @@ class DBusService final : public Daemon {
   QString getLogs();
 
  protected:
-  bool run(Op op, const Config& config) override;
+  bool run(Op op, const InterfaceConfig& config) override;
 
-  bool supportServerSwitching(const Config& config) const override;
+  bool supportServerSwitching(const InterfaceConfig& config) const override;
+  bool switchServer(const InterfaceConfig& config) override;
 
-  bool switchServer(const Config& config) override;
+  bool supportWGUtils() const override { return true; }
+  WireguardUtilsLinux* wgutils() override;
```

Comment:
This should return a `WireguardUtils` and not a `WireguardUtilsLinux`.

Reply:
Fine. I had it like this as in next PR I had added a WireguardUtilsLinux specific method